cmd/estuary-shuttle/main.go

```diff
@@ -1144,7 +1144,7 @@ func (s *Shuttle) handleAdd(c echo.Context, u *User) error {
 }
 
 func (s *Shuttle) Provide(ctx context.Context, c cid.Cid) error {
-	subCtx, cancel := context.WithTimeout(ctx, time.Second*10)
+	subCtx, cancel := context.WithTimeout(ctx, time.Second*15)
```

Comment:
This should be a variable like `provideTimeout` (after the refactor this would go into the `constants` package)